test/CMakeLists.txt

```diff
@@ -535,7 +535,7 @@ set(IMPLICITGEMM_TESTING_ENV
  MIOPEN_DEBUG_CONV_IMPLICIT_GEMM=1
 )
 
-if(WORKAROUND_ISSUE_936)
+if(WORKAROUND_ISSUE_936 AND MIOPEN_TEST_HALF)
```

Comment:
Shall we switch ON this workaround only for HALF instead? I mean, at line 45 of this file.